src/App.tsx

```diff
@@ -17,18 +55,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                searchInputValue={searchInputValue}
+                setSearchInputValue={setSearchInputValue}
```

Comment:
Consider following the naming conventions for event handler functions. According to the checklist, event handler functions should be named using the pattern `handle[EventName]` or `on[EventName]`. In this case, `setSearchInputValue` could be renamed to `handleSearchInputChange` or similar to better convey its purpose.